### PR TITLE
Addign missing Resources for System.Design* namespace.

### PR DESCRIPTION
**Pull request by dreddy-work**
We started porting APIs from System.Design* name space as per the requests we received  since winforms become open source. However, we missed porting some of the resources. As part of this change, we are making following changes

- Porting the resources required for the source code that are already ported.

- Bitmap Icons/images are replaced with Icons. Corresponding source changes were in place.

- We are porting all the images, equivalent, to desktop framework. Even though,  we did not port all the source code that require/use these images. Replacing existing Bitmaps with Icons have a cross team dependency. hence, pushing all of them in one go hoping that we fill the gap in porting the source code between core and desktop framework.

### CODE REVIEW

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/InheritanceUI.cs [hunk 1]
```diff
@@ -26,8 +26,9 @@ public Bitmap InheritanceGlyph
             {
                 if (s_inheritanceGlyph == null)
                 {
-                    s_inheritanceGlyph = new Bitmap(typeof(InheritanceUI), "InheritedGlyph.bmp");
-                    s_inheritanceGlyph.MakeTransparent();
+                    Icon iGlyph = new Icon(typeof(InheritanceUI), "InheritedGlyph");
+                    s_inheritanceGlyph = iGlyph.ToBitmap();
```
Comment (RussKie) [inline src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/InheritanceUI.cs]: in other places these two lines are combined 
```suggestion
                    s_inheritanceGlyph = new Icon(typeof(InheritanceUI), "InheritedGlyph").ToBitmap();
```

**Comment by zsd4yr** [comment]
@dreddy-work I added the xlfs you needed with the updatexlf task 😄

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/InheritanceUI.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public Bitmap InheritanceGlyph
                 if (s_inheritanceGlyph == null)
                 {
                     Icon iGlyph = new Icon(typeof(InheritanceUI), "InheritedGlyph");
```
Comment (weltkante) [inline src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/InheritanceUI.cs]: forgot to remove this line?

**Comment by RussKie** [comment]
Superseded by #1229.

**Pull request closed without merging**